apps/system/lockscreen/js/lockscreen_inputpad.js

```diff
@@ -180,10 +194,11 @@
 
       case 'c': // 'C'ancel
         this.dispatchEvent(new window.CustomEvent(
-          'lockscreen-keypad-input', { detail: {
-            key: key
-          }
-        }));
+          'lockscreen-keypad-input', {
+            detail: {
+              key: key
+            }
+          }));
```

Comment:
It appears that there is no change here except to formatting. Your formatting is better, but please don't do that if there are not actually any required changes, because it just makes it harder to review and increases the chances for merge conflicts.